rotkehlchen/utils/serialization.py

```diff
@@ -77,26 +76,24 @@ def deserialize_asset_with_oracles_from_db(
         asset_type: AssetType,
         asset_data: list[Any],
         underlying_tokens: Optional[list[UnderlyingToken]],
-        form_with_incomplete_data: bool,
 ) -> AssetWithOracles:
     """
     From a db tuple containing information about any asset deserialize to the correct Asset class
     according to type in the database.
     May raise:
     - DeserializationError
-    - UnknownAsset
     - WrongAssetType
     """
     identifier = asset_data[0]
     if asset_type == AssetType.EVM_TOKEN:
         decimals = asset_data[3]
-        if decimals is None:
-            decimals = 18
         name = asset_data[4]
         symbol = asset_data[5]
-        missing_basic_data = name is None or symbol is None
-        if missing_basic_data and form_with_incomplete_data is False:
-            raise UnknownAsset(identifier=identifier)
+
+        if decimals is None:
+            decimals = 18
+        if name is None:
```

Comment:
opening another one from this review to show you and me that this is still not done.